src/networks.js

```diff
@@ -60,7 +60,11 @@ module.exports = {
     pubKeyHash: 0x1cb8,
     scriptHash: 0x1cbd,
     wif: 0x80,
-    consensusBranchId: 0x5ba81b19,
+    consensusBranchId: {
+      1: 0x00,
```

Comment:
What do `1, 2, and 3` represent? Is there something more meaningful we can name those keys? 

```
CONSENSUS_VERSION_1 = 1
consensusBranchId: {
  CONSENSUS_VERSION_1: 0x00,
  ...
}
```

might be overkill, but from someone who has way less context trying to understand this library, it could be helpful.